Dockerfile.django

```diff
@@ -88,14 +89,21 @@ RUN \
   true
 USER root
 RUN \
-    adduser --system --no-create-home --disabled-password --gecos '' \
-        --uid ${uid} ${appuser} && \
+    # Set a known user and group with a higher number uid/git
+    addgroup --gid ${gid} ${appuser} && \
+    adduser --no-create-home --disabled-password \
+            --gecos 'DefectDojo' --uid ${uid} --gid ${gid} ${appuser} && \
     chown -R root:root /app && \
     chmod -R u+rwX,go+rX,go-w /app && \
+    # Allow for bind mounting local_settings.py and other setting overrides
+    chown root:${appuser} /app/dojo/settings && \
+    chmod 775 /app/dojo/settings && \
+    # Allow ${appuser} to run entrypoint scripts
+    chown root:defectdojo /entrypoint-* && \
+    chmod g+x /entrypoint-* && \
     mkdir /var/run/${appuser} && \
-    chown ${appuser} /var/run/${appuser} && \
-	  chmod g=u /var/run/${appuser} && \
-    mkdir -p media/threat && chown -R ${uid} media
+    chown ${appuser}:${appuser} /var/run/${appuser} && \
+    mkdir -p media/threat && chown -R ${uid}:${gid} media
```

Comment:
Can be here `chown -R ${appuser}:${appuser} media` to be consistent

Reply:
That makes sense to me.  I considered it but was trying to keep the scope limited to just the settings/bind mount problem.  I'll go ahead and do a commit to add that in.  100% agree on your thinking.